addons/sale_stock/models/res_users.py

```diff
@@ -5,16 +5,14 @@
 
 
 class Users(models.Model):
-    _inherit = ['res.users']
+    _inherit = 'res.users'
```

Comment:
Probably shouldn't do a superficial change like this in stable due to the unnecessary diff

Reply:
Changes have been implemented, thanks ;)